backend/config/urls.py

```diff
@@ -11,6 +11,7 @@
     jwt_token_refresh,
     path("admin/", admin.site.urls),
     path("api/v1/accounts/", include("apps.accounts.api.v1.routes")),
+    path("api/v1/", include("apps.vacancies.api.v1.routes")),
```

Comment:
do "api/v1/vacancies" here, so we won't have any issues with matching our urls. Even if the order is checked, it is always better to be on a safe side.